src/java-github-release.sh

```diff
@@ -7,6 +7,27 @@ throw() {
   exit 1
 }
 
+get_changelog () {
+  changelog=""
+  has_start_line=0
+  while read -r line; do
+    if [[ $has_start_line -eq 0 ]]; then
+      # If we do not have a starting line, look for one.
+      if [[ "$line" =~ \#\ \[[[:digit:]] ]]; then
+        has_start_line=1
+      fi
+    elif [[ $has_start_line -eq 1 ]]; then
+      # If we do have a starting line, look for a closing line. Either exit the loop or append to the changelog.
+      if [[ "$line" =~ \#\ \[[[:digit:]] ]]; then
+        break
+      else
+        changelog=$"$changelog\n$line"
+      fi
+    fi
+  done <CHANGELOG.md
```

Comment:
This errors if the file does not already exist.  See https://github.com/dequelabs/attest-release-scripts/issues/13